docs/getting-started/backends-and-brokers/kafka.rst

```diff
@@ -0,0 +1,82 @@
+.. _broker-kafka:
+
+=============
+ Using Kafka
+=============
+
+.. _broker-Kafka-installation:
+
+Configuration
+=============
+
+For celeryconfig.py:
+
+.. code-block:: python
+
+    import os
+
+    task_serializer = 'json'
+    broker_transport_options = {
+        # "allow_create_topics": True,
+    }
+    broker_connection_retry_on_startup = True
+
+    # For using SQLAlchemy as the backend
+    # result_backend = 'db+postgresql://postgres:example@localhost/postgres'
+
+    broker_transport_options.update({
+        "security_protocol": "SASL_SSL",
+        "sasl_mechanism": "SCRAM-SHA-512",
+    })
+    sasl_username = os.environ["SASL_USERNAME"]
+    sasl_password = os.environ["SASL_PASSWORD"]
+    broker_url = f"confluentkafka://{sasl_username}:{sasl_password}@broker:9094"
+    kafka_admin_config = {
+        "sasl.username": sasl_username,
+        "sasl.password": sasl_password,
+    }
+    kafka_common_config = {
+        "sasl.username": sasl_username,
+        "sasl.password": sasl_password,
+        "security.protocol": "SASL_SSL",
+        "sasl.mechanism": "SCRAM-SHA-512",
+        "bootstrap_servers": "broker:9094",
+    }
+
+Please note that "allow_create_topics" is needed if the topic does not exist
+yet but is not necessary otherwise.
+
+For tasks.py:
+
+.. code-block:: python
+
+    from celery import Celery
+
+    app = Celery('tasks')
+    app.config_from_object('celeryconfig')
+
+
+    @app.task
+    def add(x, y):
+        return x + y
+
+Auth
+====
+
+TODO: Document how to properly authenticate with a kafka broker that uses SASL, for example.
```

Comment:
Great new additions!
Please change PR to draft until you're done friend.

Reply:
Will do, thanks!